magpie-check-inputs

```diff
@@ -1041,6 +1042,125 @@ then
 fi
 
 #
+# Check Phoenix Inputs 
+#
+
+if [ "${PHOENIX_SETUP}" == "yes" ]
+then
+    if [ "${JAVA_HOME}X" == "X" ]
+    then
+	echo "JAVA_HOME must be set for Phoenix"
+	exit 1
+    fi
+
+    if [ "${PHOENIX_VERSION}X" == "X" ]
+    then
+	echo "PHOENIX_VERSION must be set to run Phoenix"
+	exit 1
+    fi
+
+    if [ "${PHOENIX_HBASE_VERSION}X" == "X" ]
+    then
+	echo "PHOENIX_HBASE_VERSION must be set to run Phoenix"
+	exit 1
+    fi
+
+    if [ "${PHOENIX_HOME}X" == "X" ]
+    then
+	echo "PHOENIX_HOME must be set to run Phoenix"
+	exit 1
+    fi
+    
+    if [ ! -d ${PHOENIX_HOME} ]
+    then
+	echo "PHOENIX_HOME does not point to a directory"
+	exit 1
+    fi
+
+    if [ "${PHOENIX_LOCAL_DIR}X" == "X" ]
+    then
+	echo "PHOENIX_LOCAL_DIR must be set to run Phoenix"
+	exit 1
+    fi
+
+    if [ "${HBASE_SETUP}" != "yes" ]
+    then
+	echo "HBase must be enabled for Phoenix"
+	exit 1
+    fi
+
+    if [ "${ZOOKEEPER_SETUP}" != "yes" ]
+    then
+	echo "Zookeeper must be enabled for Hbase"
+	exit 1
+    fi
```

Comment:
this isn't necessary b/c it's checked w/n hbase right?  And we already checked for hbase_setup == yes?